app/server/views/styles/components/slide.scss

```diff
@@ -113,7 +113,7 @@
    ========================================================================== */
 
 /* Title slide is a bigger size than video to go to a video file. So we need to scale all the sizes up */
-/* 2022 Oct sizing for carolus is height: 602px, width: 1072px.*/
+/* 2022 Oct sizing for carolus is height: 588px, width: 1036px.*/
```

Comment:
The height for video is not changed for module.
As I am not aware of exactly what goes in this, does main container width play a role in deciding what width and height should be?